lms/static/sass/shared-v2/_components.scss

```diff
@@ -0,0 +1,49 @@
+// TODO: tabs should be added to the Pattern Library
+.tabs {
+    @include clearfix();
+    @extend %reset-lists;
+    @include border-top-radius(4px);
+    padding: ($baseline*0.75) 0 ($baseline*0.75) 0;
+
+    .tab {
+        @include float(left);
+        list-style: none;
+        margin-bottom: 0;
+
+        &.prominent {
+            @include margin-right(16px);
+            background: rgba(255, 255, 255, 0.5);
+            border-radius: 3px;
+        }
+
+        &.prominent + li {
+            @include border-left(1px solid $lms-border-color);
+            padding-left: $baseline*0.75;
+        }
+
+        a,
+        a:visited {
+            padding: ($baseline/2) ($baseline*0.75) 13px ($baseline*0.75);
+            display: block;
+            text-align: center;
+            text-decoration: none;
+            border-style: solid;
+            border-width: 0 0 4px 0;
+            border-bottom-color: transparent;
+            color: $lms-label-color;
+            font-size: 14px;
+
+            &:hover,
+            &:focus {
```

Comment:
Some inconsistency on whether we state `.foo, .bar` selectors one per line or on the same line (see [here](https://github.com/edx/edx-platform/pull/12538/files#diff-c49c4f432d08c45862665db64fe1ff6bR204) for example). I like one per line slightly more but don't particularly mind either, we should just pick one and stick to it.